test/clj/game/cards/events_test.clj

```diff
@@ -4484,10 +4603,10 @@
       (is (nil? (get-run)) "Run is ended")))
   (testing "with Gauntlet #2942"
     (do-game
-      (new-game {:corp {:deck [(qty "Hedge Fund" 5)]
-                        :hand [(qty "Vanilla" 5)]
-                        :credits 10}
-                 :runner {:hand ["The Gauntlet" "Corroder" "Rip Deal"]
+      (new-game {:corp   {:deck    [(qty "Hedge Fund" 5)]
```

Comment:
same with the spacing here

Reply:
appears I need to figure out how to edit the cursive ide autoformatter.